public/stylesheets/site/2.0/26-media-narrow.css

```diff
@@ -182,7 +182,10 @@ body .narrow-shown {
   padding: 0;
 }
 
-.splash div.module, .logged-in .splash div.module {
```

Comment:
Could you put this back on one line like the other selectors, please? (Yeah, it's easier to see a diff if it's already spread across lines, but let's go for consistency until we do a bulk change.)